### PR TITLE
Enable force option for tagging

### PR DESCRIPTION
**Pull request by poznachowski**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -225,13 +231,15 @@
...
               <failsOnError>true</failsOnError>
-              <linkXRef>false</linkXRef>
             </configuration>
             <goals>
               <goal>check</goal>
             </goals>
           </execution>
         </executions>
+        <configuration>
+          <linkXRef>false</linkXRef>
```
Comment (davidxia) [inline pom.xml]: Just wondering why we don't want this. I know we have this in our other helios projects, but I don't know why. I see [docs here](https://maven.apache.org/plugins/maven-checkstyle-plugin/checkstyle-mojo.html) say

> Link the violation line numbers to the source xref. Will link automatically if Maven JXR plugin is being used

File: pom.xml [hunk 2]
```diff
@@ -115,7 +115,13 @@
     <dependency>
       <groupId>com.spotify</groupId>
       <artifactId>docker-client</artifactId>
-      <version>2.7.6</version>
+      <version>2.7.20</version>
+    </dependency>
+    <dependency>
+      <groupId>com.spotify</groupId>
```
Comment (davidxia) [inline pom.xml]: Why do we need the same dependency shaded?

**Comment by davidxia** [comment]
@poznachowski Looks great. I just had a few questions above.

File: pom.xml (hunk 1 above)
Comment (poznachowski) [inline pom.xml]: I did not change anything in the configuration, this prop was already there:
https://github.com/spotify/docker-maven-plugin/blob/master/pom.xml#L228
I just moved it outside of the execution step as my IDE was complaining that is was against validate phase configuration schema.

**Comment by poznachowski** [comment]
I tried to squash, but it seems I failed hardly.. :) Could you help me out here? (I never squashed before)

**Comment by davidxia** [comment]
@poznachowski Sure thing.

Interactively rebase last 3 commits with `git rebase -i HEAD~3`. You'll see three lines at the top with "pick." Change the last two to "s" for squash. Save and quit. Reword your commit message. Save and quit. `git log` will show you one commit now.

**Comment by poznachowski** [comment]
That's exactly what I did, let me try once more.

**Comment by davidxia** [comment]
closed by https://github.com/spotify/docker-maven-plugin/commit/ff8321980ce492fed39bdd6cfdc7b5e4e79a5c08

**Pull request closed without merging**